### ISSUE TITLE
question about fig 5.6

### ISSUE THREAD

**Issue by zeuschaeng** [OHDSI/TheBookOfOhdsi#114]
In fig 5.6, Fibrillation is represented as a child of Supraventricular arrhythmia

![](http://support-forum.s3.dualstack.ap-northeast-2.amazonaws.com/original/1X/8463826c9de678b293daea3ac683b87271710664.png)

But, when you search for Fibrillation in ATLAS, you can find it is not.

![](http://support-forum.s3.dualstack.ap-northeast-2.amazonaws.com/original/1X/1f79646a1d82b53ff5749f8edf2b2ed0b556b095.png)

**Comment by schuemie** [comment]
Yes, you're right. This seems to have been a correction in the vocabulary, because fibrillation isn't necessary a supraventricular arrhythmia (it could also be a ventricular fibrillation).

We'll need to update this example in the book. Thanks!

**Comment by cgreich** [comment]
Oh! Pain!!!!! We have to change it in all the Tutorial and everywhere.

**Comment by zeuschaeng** [comment]
Sorry. I am too late.
